### PR TITLE
Adding historic Board members (Silona, Yao) + Adding 2022 Officers

### PR DESCRIPTION
**Pull request by spier**
Two distinct changes:

1) Adding SIlona and Yao to the list of historic Board members as requested.
2) Also adding 2022 Officers - see #283

Sources for (1):
- bio + pic / https://opensource101.com/speakers/silona-bonewald/
- bio / https://www.oreilly.com/pub/e/3884
- pic / https://github.com/NewMexicoKid

Follow up to #266.

### CODE REVIEW

**Comment by rrrutledge** [comment]
Recommend https://www.oreilly.com/pub/au/7440 for @NewMexicoKid picture

**Comment by spier** [comment]
> Recommend https://www.oreilly.com/pub/au/7440 for @NewMexicoKid picture

Why? Just curious what expectations we have for Board member pictures.

**Review by dicortazar — APPROVED** [review]
This looks good to me!

However, I do not usually review the website commits. We may want to have another reviewer, just in case :).

**Comment by gruetter** [comment]
Max is no longer working at the Research group mentioned in the Bio! This needs an update. @maxcapraro : could you provide an updated bio?

File: content/about/board/_index.md [hunk 1]
```diff
@@ -81,3 +81,11 @@ Russ's drive and passion is to enable all software engineers to achieve incredib
...
 {{< /board-member >}}
+
+{{< board-member name="Tim Yao" image="/images/about/Tim_Yao.jpeg" >}}
+Tim is a Distinguished Member of Technical Staff and Domain Leader for Open Source Innovation in the Nokia Mobile Network CTO Core and Cloud Innovation team, working on software reuse, InnerSource, open source and cloud robotics topics. Over his nineteen-year career in telecommunications, Tim has held roles in software testing, system engineering, architecture, and procurement. He served six years on the Alcatel-Lucent FOSS Executive Committee. Tim has experience creating grass roots communities within the company and outside of it; he is a volunteer co-Municipal Liaison for National Novel Writing Month.
```
Comment (NewMexicoKid) [inline content/about/board/_index.md]: Updated bio:
Tim is a Distinguished Member of Technical Staff and the Product Manager for Nokia Continuous Delivery in Nokia Cloud and Network Services. Over his twenty-plus year career in telecommunications, Tim has held roles in innovation, software testing, system engineering, architecture, and procurement. He served six years on the Alcatel-Lucent FOSS Executive Committee. Tim has experience creating grass roots communities within the company and outside of it; he is a volunteer co-Municipal Liaison for National Novel Writing Month.

File: content/about/board/_index.md [hunk 2]
```diff
@@ -81,3 +81,11 @@ Russ's drive and passion is to enable all software engineers to achieve incredib
 {{< board-member name="Maximilian Capraro" image="/images/about/Maximilian_Capraro.jpg" >}}
 Maximilian Capraro is a researcher and doctoral candidate at the Open Source Research Group at Friedrich-Alexander-University Erlangen-Nürnberg. He worked on inner source in research projects with a variety of companies including Black Duck Software, Continental, and multiple Siemens business units. He developed the patch-flow measurement method (for measuring software development collaboration) and the first classification framework for inner source projects and programs. His research interests the quantification of software development collaboration, inner source governance including legal and taxation aspects, and many other things around inner source, open source, and software engineering.
 {{< /board-member >}}
+
+{{< board-member name="Tim Yao" image="/images/about/Tim_Yao.jpeg" >}}
```
Comment (NewMexicoKid) [inline content/about/board/_index.md:85]: Suggest using my portrait at https://avatars.githubusercontent.com/u/2311408?v=4

**Review by NewMexicoKid — DISMISSED** [review]
I've suggested updates for my bio and photo. Thanks for doing this.

File: content/about/board/_index.md (hunk 1 above)
Comment (spier) [reply content/about/board/_index.md]: Updated in [fcc2ba6](https://github.com/InnerSourceCommons/innersourcecommons.org/pull/278/commits/fcc2ba6efdde60fcdac1bbba6777fade2cb151cb)

File: content/about/board/_index.md (hunk 2 above)
Comment (spier) [reply content/about/board/_index.md:85]: Ah, nice one!
Updated in [6231379](https://github.com/InnerSourceCommons/innersourcecommons.org/pull/278/commits/62313790451b5ffd7e46d02b6e63540497404ec3)

**Comment by spier** [comment]
> Max is no longer working at the Research group mentioned in the Bio! This needs an update. @maxcapraro : could you provide an updated bio?

We need to get this PR out, so that we can list the new Officers on the Board page.
@maxcapraro please open a new PR to update your bio if required.

**Review by rrrutledge — DISMISSED** [review]
Thanks for this work, @spier ❗   I opened a pull request to this branch to [Add assistant officer titles](https://github.com/InnerSourceCommons/innersourcecommons.org/pull/284).

**Review by clcoffey — APPROVED** [review]
From the website perspective, it looks good to me however I didn't check the text.

**Pull request merged**